ansible_runner/runner_config.py

```diff
@@ -178,6 +180,18 @@ def prepare(self):
             self.env['ANSIBLE_STDOUT_CALLBACK'] = 'minimal'
         else:
             self.env['ANSIBLE_STDOUT_CALLBACK'] = 'awx_display'
+
+            if self.resource_profiling:
+                callback_whitelist = os.environ.get('ANSIBLE_CALLBACK_WHITELIST', '').strip()
+                self.env['ANSIBLE_CALLBACK_WHITELIST'] = callback_whitelist + (',' if len(callback_whitelist) else '') + 'cgroup_perf_recap'
+                self.env['CGROUP_CONTROL_GROUP'] = '{}/{}'.format(self.resource_profiling_base_cgroup, self.ident)
+                self.env['CGROUP_OUTPUT_DIR'] = self.private_data_dir + '/profiling_data'
+                self.env['CGROUP_FILE_NAME_FORMAT'] = '%(task_uuid)s-%(feature)s.%(ext)s'
+                self.env['CGROUP_OUTPUT_FORMAT'] = 'json'
+                self.env['CGROUP_CPU_POLL_INTERVAL'] = '0.25'  # TODO
+                self.env['CGROUP_FILE_PER_TASK'] = 'True'
+                self.env['CGROUP_WRITE_FILES'] = 'True'
+                self.env['CGROUP_DISPLAY_RECAP'] = 'False'  # TODO
```

Comment:
Does this just amount to an extra verbose event?  How does this materialize if you set it to `True`?

Reply:
You get a summary at the end, like this:
![Screenshot 2019-03-07 09 33 36](https://user-images.githubusercontent.com/4440360/55581672-4e66ed80-56d2-11e9-8886-69bb65d6bd83.png)

Seems like we would want to make this configureable, too - what do you think?

Reply:
Is the summary part of the `playbook_on_stats` event or a verbose event at the end?

Reply:
The information you're seeing is just [printed](https://github.com/ansible/ansible/blob/f0480ab132bc2beb1feea42c7fcd714429114f38/lib/ansible/plugins/callback/cgroup_perf_recap.py#L414-L415) by the plugin:

![image](https://user-images.githubusercontent.com/4440360/55593841-41f28d00-56f2-11e9-9a2b-e28b5d5de909.png)

If we wanted to collect this information from the plugin, we'd need to scrape / parse the information that gets printed. (The actual profiling data gets written, as json, to a set of files which we collect at the end of each task)

Reply:
If it gets printed (i.e... emitted to `stdout`) then we'll pick them up as `verbose` events. We should check to see if these are coming in as verbose events (would be good to know)